package.json

```diff
@@ -21,7 +21,10 @@
     "react-feather": "^2.0.8",
     "tailwindcss": "^1.9.6",
     "twin.macro": "^1.12.1",
-    "validator": "^13.1.17"
+    "validator": "^13.1.17",
+    "react-router": "^5.1.2",
+    "react-router-dom": "^5.1.2",
+    "react-router-hash-link": "^1.2.2"
```

Comment:
since exporting `NavLink` and sharing them in application end is not gonna work, omit react-router stuff?

Reply:
I need this for the storybook and test